src/client/user/components/Drawer/ControlPanel/LinkAnalytics/widgets/DailyStatistics/DownloadClicksButton.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { saveAs } from 'file-saver'
+import Typography from '@material-ui/core/Typography'
+import makeStyles from '@material-ui/core/styles/makeStyles'
+import rootActions from '../../../../../../../app/components/pages/RootPage/actions'
+import downloadIcon from '../../assets/download.svg'
+import { get } from '../../../../../../../app/util/requests'
+import useIsIE from '../../../../../../../app/components/BaseLayout/util/ie'
+import { useDrawerState } from '../../../../index'
+
+async function downloadClicks(shortUrl: string, onError: () => void) {
+  const offsetDays = 3650
```

Comment:
ten years of statistics? is that going to be okay?

Reply:
discussed that the server presently has no issues with the amount of data to fetch and should not be an issue for awhile, at least until we allow more control over the period of data to download.